root/usr/bin/nsdc-cmd

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2018 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+# Execute a command inside the containter
+
+. /usr/share/nethserver-dc/nsdc-functions
+
+function help
+{
+    echo "Usage: $0 [-r] <cmd> [<args1> ... <argsN>]"
+    echo "  -r : execute raw command - use it for reading from stdin"
+    exit 1
+}
+
+raw=0
+while getopts ":h:r" opt; do
+  case $opt in
+    h)
+      help
+      ;;
+    r)
+      raw=1
+      shift
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      ;;
+  esac
+done
+
+if [ $# -eq 0 ]; then
+    help
+fi
+
+if [ $raw -eq 1 ]; then
+    cmd_raw $@
```

Comment:
Use the preserve whitespace syntax

      cmd_raw "$@"

Reply:
I already tested it, but it doesn't really change much.
Whitespace is already preserved but quotes are not and I didn't find a good way to do it.

But I will add the quotes, for a better implementation.